app/src/test/java/com/eveningoutpost/dexdrip/Models/BlueReaderTest.java

```diff
@@ -486,6 +490,25 @@ public void processNewTransmitterData_NormalBg_LowBattery() {
         assertThat(lastBgReading.filtered_data).isEqualTo(150.0d);
```

Comment:
as above. Please also have a look at https://google.github.io/truth/ to get more info on how to utilize google truth :-)